assignments/lambda-classes.js

```diff
@@ -1 +1,149 @@
+		return newGrade;
+  }
+  }
+ 
+
+//   #### Student
+
+//   * Now we need some students!
+//   * Student uses the same attributes that have been set up by Person
+//   * Student has the following unique props:
+//   * `previousBackground` i.e. what the Student used to do before Lambda School
+//   * `className` i.e. CS132
+//   * `favSubjects`. i.e. an array of the student's favorite subjects ['Html', 'CSS', 'JavaScript']
+//   * Student has the following methods:
+//   * `listsSubjects` a method that logs out all of the student's favoriteSubjects one by one.
+//   * `PRAssignment` a method that receives a subject as an argument and logs out that the `student.name has submitted a PR for {subject}`
+//   * `sprintChallenge` similar to PRAssignment but logs out `student.name has begun sprint challenge on {subject}`
+  
+
+  class Student extends Person {
+    constructor(data){
+        super(data)
+        this.className = data.className
+        this.favSubjects = data.favSubjects
+        this.previousBackground = data.previousBackground
+
+        this.grade = (Math.random() * 100).toFixed(0);
+        this.projectManager = projectManager
+  }
+
+  listsSubjects(){
+       console.log(`${this.favSubjects}`);
+       return `${this.favSubjects}`;
+  }
+
+  PRAssignment(subject){
+    console.log(`${student.name} has submitted a PR for ${subject}`);
+    return `${this.favSubjects}`;
+  }
+
+  sprintChallenge(subject){
+    console.log(`${student.name} has begun sprint challenge on ${subject}`);
+    return `${student.name} has begun sprint challenge on ${subject}`;
+    
+  }
+
+  graduate(){
+      while(this.grade<70) {
+          this.grade = this.projectManager.gradeStudent(this);
+      }
+      console.log(`${this.name} just graduated!`);
+      return `${this.name} just graduated!`;
+  }
+
+  }
+//   ### Project Manager
+
+// * Now that we have instructors and students, we'd be nowhere without our PM's
+// * ProjectManagers are extensions of Instructors
+// * ProjectManagers have the following unique props:
+// * `gradClassName`: i.e. CS1
+// * `favInstructor`: i.e. Sean
+// * ProjectManagers have the following Methods:
+// * `standUp` a method that takes in a slack channel and logs `{name} announces to {channel}, @channel standy times!​​​​​
+// * `debugsCode` a method that takes in a student object and a subject and logs out `{name} debugs {student.name}'s code on {subject}`
+
+class ProjectManagers extends Instructor{
+    constructor(data){
+        super(data)
+        this.gradClassName = data.gradClassName
+        this.favInstructor = data.favInstructor
+}
+
+standUp(name, channel){
+    console.log(`${name} announces to ${channel}, @channel standy times!​​​​​`)
+    return `${name} announces to ${channel}, @channel standy times!​​​​​`
+    
+}
+
+debugsCode(){
+    console.log(`${name} debugs ${student.name}'s code on ${subject}`)
+    return `${name} debugs ${student.name}'s code on ${subject}`
+}
+}
+
+
+/*
+
```

Comment:
I would have liked to see a couple of instances of student, instructor and PM in addition to console logging some methods.